src/main/java/com/orbitz/consul/cache/ConsulCache.java

```diff
@@ -202,6 +202,14 @@ public boolean addListener(Listener<K, V> listener) {
         return added;
     }
 
+    public Iterable<Listener<K, V>> getListeners() {
```

Comment:
Sure, I can change that and even remove the count method, as we can expose the listeners as a list, not `Iterable`.